src/shared/analytics/shared/mixpanel-data-helpers.ts

```diff
@@ -1,10 +1,9 @@
-import { isTruthy } from 'is-truthy-ts';
 import type { Account } from 'src/background/account/Account';
 import { getAddressActivity } from 'src/ui/shared/requests/useAddressActivity';
 import { INTERNAL_SYMBOL_CONTEXT } from 'src/background/Wallet/Wallet';
 import { isReadonlyContainer } from 'src/shared/types/validators';
-import { ZerionAPI } from 'src/modules/zerion-api/zerion-api.background';
 import { backgroundQueryClient } from 'src/modules/query-client/query-client.background';
+import { getWalletsMetaByChunks } from 'src/modules/zerion-api/requests/wallet-get-meta';
```

Comment:
But here you're supposed to use `ZerionAPI.getWalletsMetaByChunks`
Make sure to use `zerion-api.background`